dashboard/config/routes.rb

```diff
@@ -60,7 +60,8 @@ module OPS
 
   get 'redirected_url', to: 'redirect_proxy#get', format: false
 
-  get 'docs/*docs_route', to: 'docs_proxy#get'
+  get 'docs/*docs_path', to: 'curriculum_proxy#get'
+  get 'curriculum/*curriculum_path', to: 'curriculum_proxy#get'
```

Comment:
Now that I'm looking at this more, would it make sense to have these be 2 separate actions (that can each call into a common private/protected method)? Then you won't need to even inspect and branch on the request path, and it might make testing easier.

Reply:
It could. It felt a little weird to have two separate actions for what's really the same thing, but given that I end up branching right away inside the action anyways, perhaps that does make more sense - and does probably make this easier to reason about.